### DISCUSSION TITLE
有没有办法针对群单独设置一组直播间和主播提醒？

### DISCUSSION THREAD

**Discussion by leecoooo** [eric2788/vup_monitors#30] [Q&A] [answered]
也就是这组直播间和主播相对其他群是透明的，不会互相影响？

**Comment by eric2788** [comment]
高亮名单每个群都是不一样的吧
如果你是指开播通知，那可以加个群注视名单

**Comment by leecoooo** [comment]
> 高亮名单每个群都是不一样的吧 如果你是指开播通知，那可以加个群注视名单

高亮确实不一样，但是多个群有同一个高亮用户的话，都会同时间提醒，我想实现的是同一个高亮用户在不同群提醒的直播间不一样。

还有那个注视名单是怎么理解，一直不太理解。

**Comment by eric2788** [comment]
你試試為每個群添加不同的注視名單？例如 vtuber A 的群注視只添加 vtuber A, vtuber B 的群注視只添加 vtuber B

**Comment by Marxel-cn** [comment, marked as answer]
> > 高亮名单每个群都是不一样的吧 如果你是指开播通知，那可以加个群注视名单
> 
> 高亮确实不一样，但是多个群有同一个高亮用户的话，都会同时间提醒，我想实现的是同一个高亮用户在不同群提醒的直播间不一样。
> 
> 还有那个注视名单是怎么理解，一直不太理解。

要不要试试用两个bot实例？（不过需要develop分支我提交的新代码支持）
设置两个bot使用不同的id连接同一个biligo-live-ws，然后在go-cqhttp上开放两个端口分别给两个bot
两个bot使用不同的直播间监控列表（subscribing）即可
（比如botA高亮P家主播，监控P家直播间，botB高亮所有日本主播，监控propro直播间之类）
